datatypes/src/dataset.rs

```diff
@@ -64,3 +61,297 @@ impl From<ExternalDataId> for DataId {
         DataId::External(value)
     }
 }
+
+pub const NAME_DELIMITER: char = ':';
+pub const SYSTEM_NAMESPACE: &str = "_";
+pub const SYSTEM_PROVIDER: &str = "_";
+
+/// The user-facing identifier for loadable data.
+/// It can be resolved into a [`DataId`].
+#[derive(Debug, Clone, Hash, Eq, PartialEq)]
+pub struct NamedData {
```

Comment:
Can you maybe briefly explain how the named data and workspaces and providers work? Should also be in the docs then

Reply:
[6e97ae8](https://github.com/geo-engine/geoengine/pull/782/commits/6e97ae81f0f45f0fd46f095e57ae0b6238f9e15d)